gobblin-service/src/main/java/org/apache/gobblin/service/modules/spec/JobExecutionPlanDagFactory.java

```diff
@@ -80,6 +83,7 @@ public Dag<JobExecutionPlan> createDag(List<JobExecutionPlan> jobExecutionPlans)
       }
     }
     Dag<JobExecutionPlan> dag = new Dag<>(dagNodeList);
+    log.info("Multi-hop dag created with id {} and jobs: {}", DagManagerUtils.generateDagId(dag), jobNames);
```

Comment:
Let's not add Multi-hop here because this also includes single hop flows with single jobs